MediaBrowser.MediaEncoding/Probing/ProbeResultNormalizer.cs

```diff
@@ -1338,6 +1338,10 @@ private void SetAudioInfoFromTags(MediaInfo audio, Dictionary<string, string> ta
             mb = GetMultipleMusicBrainzId(tags.GetValueOrDefault("MusicBrainz Release Track Id"))
                  ?? GetMultipleMusicBrainzId(tags.GetValueOrDefault("MUSICBRAINZ_RELEASETRACKID"));
             audio.TrySetProviderId(MetadataProvider.MusicBrainzTrack, mb);
+
+            mb = GetMultipleMusicBrainzId(tags.GetValueOrDefault("MusicBrainz Track Id"))
```

Comment:
I'm not a big fan of adding more tag parsing ability to the now (deprecated) ffprobe based parser, but I'm also fine with this if it does not have practical downsides.

Reply:
This has been written before I knew this probing will be deprecated, so now it's kind of just being left here. I don't have any strong opinions either. 

As for usefulness... If we assume most users are using Picard for tagging, then ffmpeg can't even read the tag, so it's not going to be any better here. In that case I'd be for removing it. But I have no statistics on that, so I don't know.